src/modules/battery.cpp

```diff
@@ -275,24 +334,36 @@ const std::tuple<uint8_t, float, std::string, float> waybar::modules::Battery::g
     }
 
     return {cap, time_remaining, status, total_power / 1e6};
+#endif
   } catch (const std::exception& e) {
     spdlog::error("Battery: {}", e.what());
     return {0, 0, "Unknown", 0};
   }
 }
 
 const std::string waybar::modules::Battery::getAdapterStatus(uint8_t capacity) const {
+#if defined(__Linux__)
```

Comment:
Should be lowercase:
```suggestion
#if defined(__linux__)
```